addons/mrp_subcontracting_dropshipping/models/stock_picking.py

```diff
@@ -31,7 +31,14 @@ def _action_done(self):
             if not dropship_svls:
                 continue
 
-            subcontract_svls = move.move_orig_ids.stock_valuation_layer_ids
+            if any(move.move_orig_ids.production_id.mapped('backorder_sequence')):
```

Comment:
I would add a comment to explain why we need to do that because it is not easy to figure it out, and a comment is faster than going on the PR.
Something like
> In case there is backorder(s), use only SVL from the last backorder's move to
avoid to compensate already compensated SVL.